### ISSUE TITLE
Build for LinuxDevices is creating DuetSoftwareFramework/src/LinuxDevices/C:/Users/... 

### ISSUE THREAD

**Issue by gtjoseph** [chrishamm/DuetSoftwareFramework#81]
The full path:
DuetSoftwareFramework/src/LinuxDevices/C:/Users/Christian/Source/repos/DuetSoftwareFramework/src/LinuxDevices

**Comment by gtjoseph** [comment]
Oh, it's also creating pkg/package-lock.json

**Comment by chrishamm** [comment]
The latest version of DSF doesn't generate any of those paths on my machines. Is there any way to reproduce this?

**Comment by gtjoseph** [comment]
Linux vs Windows?
Happens every time for me just doing a plain...
```
$ ./build.sh --target-arch=armv7h --build-type=Release --packages=progs --no-pkgs deb
$ git status
On branch dev
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	../src/LinuxDevices/C:/

```

**Comment by chrishamm** [comment]
Ah yes, it's generated every time the LinuxDevices library is built with the Release configuration. I'll fix it - thanks for report!

**Comment by chrishamm** [comment]
This was fixed in DSF 1.2.1.
